### PR TITLE
Marvin is now packaged and locally installed

### PR DESCRIPTION
**Pull request by eliasgomes**

### CODE REVIEW

File: helper_scripts/cloudstack/setup_cloudstackOps.sh [hunk 1]
```diff
@@ -1,38 +0,0 @@
-#!/bin/bash
```
Comment (remibergsma) [inline helper_scripts/cloudstack/setup_cloudstackOps.sh]: Please keep this file as it is used for CloudStackOps toolbox. Not related to Marvin change.

File: helper_scripts/cosmic/setup_cloudstackOps.sh [hunk 2]
```diff
@@ -1,39 +0,0 @@
-#!/bin/bash
```
Comment (remibergsma) [inline helper_scripts/cosmic/setup_cloudstackOps.sh:1]: Please keep this file as it is used for CloudStackOps toolbox. Not related to Marvin change.

Comment (eliasgomes) [reply helper_scripts/cosmic/setup_cloudstackOps.sh:1]: I don't know what you're talking about :P

**Comment by remibergsma** [comment]
Can be merged after https://github.com/MissionCriticalCloud/cosmic/pull/212

**Pull request merged**